platform/testing/end-to-end/automatic-cypress-viewport-updater.md

```diff
@@ -0,0 +1,59 @@
+# Cypress Viewport Updater Bot
+
+The Automatic Cypress Viewport Updater updates the following files in `vets-website` each month with recent data from Google Analytics:
+
+- `config/cypress.json`
+- `src/platform/testing/e2e/cypress/support/commands/viewportPreset.js`
+
+The updater:
+
+- Requests two Google Analytics reports from the `ga.gov` organization:
```

Comment:
Should this be the `va.gov` organization?

Reply:
Um... yeah!?! haha, thanks for catching that typo!